mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -231,9 +237,12 @@ def __statistics_overview(self, series, col):
             )
 
         # Detect mismatched formats for some column types
-        invalid_rows = get_mismatched_rows(series_non_null, column_type)
-        data[f'{col}/invalid_value_count'] = len(invalid_rows)
-        data[f'{col}/invalid_values'] = invalid_rows[:INVALID_VALUE_SAMPLE_COUNT]
+        invalid_rows = find_syntax_errors(series_non_null, column_type)
+        data[f'{col}/invalid_value_count'] = invalid_rows.sum()
+        invalid_values = series_non_null[invalid_rows]
+        data[f'{col}/invalid_values'] = invalid_values[:INVALID_VALUE_SAMPLE_COUNT].tolist()
+        invalid_indices = series.index.get_indexer(invalid_values.index)
```

Comment:
is it needed? we didn't reindex the series after filtering null values. we can just use `invalid_values.index`

Reply:
nvm, just saw your latest comment.